content/projects/data-science-specific/nobel-prize-winners/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: A Visual History of Nobel Prize Winners
```

Comment:
title, not Title.

Not sure if you linted this. If the linter passed then it means it's broken

Reply:
forgot to lint... my bad